apis/alarm/src/lib.rs

```diff
@@ -63,7 +63,7 @@ impl<S: Syscalls, C: platform::subscribe::Config> Alarm<S, C> {
     /// Returns number of concurrent notifications supported,
     /// 0 if unbounded.
```

Comment:
We probably want to remove this doc-comment as well then. I don’t know where it originated, I looked into the alarm driver in the kernel, as well as its system call documentation. Neither specify anything along these lines.

Reply:
Thanks.

Reply:
Those came from the old alarm system call docs, which used to support that. Thanks for fixing.